docs/04_For_Developers/06_Project_Maintainer_Guide.md

```diff
@@ -0,0 +1,58 @@
+# Project Maintainer Guide
+RSS-Bridge project maintainers are responsible for:
+1. Tagging issues
+1. Reviewing and merging pull requests
+1. Creating releases
+
+Project maintainers should keep all communications related to the project open
+to the community (Preferably GitHub issues but IRC/Matrix is also allowed).
+
+## Issues
+Maintainers should triage and respond to issues if possible, making sure to
+specify the tag if possible. For example, the 'Bug Report' can be clarified to
+'Bridge-Broken' once that is established. Project maintainers are encouraged to
+mention the bridge maintainer to comment on issues related to a specific bridge.
+
+An issue should remain open until it is resolved by fixing the issue or adding
+the feature it describes. There are some exceptions:
+- In the case of a feature request that will not be added, the issue can be
+  closed with the 'wontfix' label after pinging other active maintainers.
+- If an issue is too obscure or unclear and the issue creator does not respond
+  to additional clarification questions for 1 month, these issues should be
+  closed as they cannot be acted upon.
+
+## Pull Requests
+Maintainers should thoroughly **review** and **test** pull requests before
+merging them. Most bridge authors use other bridges in the project for reference
+so it is important to maintain a high quality codebase so that it does not
+deteriorate.
+
+To merge a pull request, use the "Squash and Merge" option in GitHub to compress
+the commits to a single change. When doing this, remove the extra commit
+description. However, if multiple authors contribute to a pull request, do not
+remove the Co-authored-by lines at the end of the message.
+
+Maintainers must not push to master, but create pull requests instead.
+Maintainers should not merge their own pull requests except in the following
+cases:
+- The pull request received no reviews or comments for 2 weeks.
+- The pull request is a trivial change. This definition is a little flexible,
+  but the diff should not exceed 5 lines and typically targets only a single
+  bridge.
+
+## Releases
+Releases are created about every 6 months, though this schedule can be
+accelerated. Maintainers should begin the process by discussing with other
+active maintainers what needs to be done before the release is published. If
+desired, these can be tracked through a GitHub issue or milestone.
+
+Release versions follow the scheme YYYY-mm-dd. Once a consensus is reached that
+a release should be published, follow these steps:
+1. Save the release notes to a draft release, following the steps in
+   contrib/prepare_release/rssbridge-log-helper.el
```

Comment:
`rssbridge-log-helper.el` is an emacs lisp script. Is it a good idea to include that in a release process? Should all maintainers learn to use emacs in order to create a release?

Reply:
No knowledge of emacs is required to use the script since only very basic features are used and step-by-step instructions are included in the document. Regardless, this script is only a helper. Feel free to contribute a vim/vscode/etc plugin that does the same thing, which is just converting a commit log into specific readable categories.

I updated the instructions in rssbridge-log-helper.el to be more thorough and explained this script in this document.

Reply:
https://github.com/RSS-Bridge/rss-bridge/pull/2819/files#diff-6249640c10194adfde731ff95f835613a0e21a44171b0af7f2e085d5fcafbf88R9-R25